crates/uv-trampoline/.cargo/config.toml

```diff
@@ -0,0 +1,3 @@
+[unstable]
+build-std = ["core", "panic_abort", "alloc", "std"]
+build-std-features = ["compiler-builtins-mem"]
```

Comment:
Did you mean to include this in this PR?

Reply:
Yes that's intentional, considering that I won't land my `std` branch anytime soon. It requires me to only type `cargo build --release --target x86_64-pc-windows-msvc` instead of that plus the `-Z` compiler flags.